src/config/remote-config.c

```diff
@@ -238,6 +241,16 @@ int Read_Remote(XML_NODE node, void *d1, __attribute__((unused)) void *d2)
 
             logr->rids_closing_time = (int) rids_closing_time;
 
+        } else if (strcmp(node[i]->element, xml_tcp) == 0) {
```

Comment:
Currently, other options related to the connection are directly under `<remote>`. Please set `<connection_overtake_time>` under `<remote>` and remove `<tcp>`.

Reply:
Done!